public/js/controllers/app_controller.js

```diff
@@ -75,6 +75,13 @@ class Controller {
                 module.TrafficMetricsController.updateTotals
             )
         });
+        // Update Africa's Talking balance
```

Comment:
can we import traffic_metrics_controller.js once in l72 and load all the relevant modules.

Reply:
yes, thanks.